base16-theme.el

```diff
@@ -773,6 +773,17 @@ return the actual color value.  Otherwise return the value unchanged."
      (slime-repl-result-face                       :foreground base0B)
      (slime-repl-output-face                       :foreground base0D :background base01)
 
+;;;; smart mode line
```

Comment:
This should match the package name (include the dashes)

Reply:
Fixed.